<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>834</width>
    <height>772</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>541</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>어떤 영화를 좋아하세요?</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_recommendation">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>731</width>
     <height>411</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_keyword">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>661</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_recommendation">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>230</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>추천</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
